sign/dilithium/mode3/internal/sample_test.go

```diff
@@ -263,3 +263,91 @@ func TestDeriveUniformB60(t *testing.T) {
 		}
 	}
 }
+
+func TestDeriveUniformX4(t *testing.T) {
+	if !DeriveX4Available {
+		t.SkipNow()
+	}
+	var ps [4]common.Poly
+	var p common.Poly
+	var seed [32]byte
+	nonces := [4]uint16{12345, 54321, 13532, 37377}
+
+	for i := 0; i < len(seed); i++ {
+		seed[i] = byte(i)
+	}
+
+	PolyDeriveUniformX4([4]*common.Poly{&ps[0], &ps[1], &ps[2], &ps[3]}, &seed,
+		nonces)
+	for i := 0; i < 4; i++ {
+		PolyDeriveUniform(&p, &seed, nonces[i])
+		if ps[i] != p {
+			t.Fatal()
+		}
+	}
+}
+
+func TestDeriveUniformLeGamma1X4(t *testing.T) {
+	if !DeriveX4Available {
+		t.SkipNow()
+	}
+	var ps [4]common.Poly
+	var p common.Poly
+	var seed [48]byte
+	nonces := [4]uint16{12345, 54321, 13532, 37377}
+
+	for i := 0; i < len(seed); i++ {
+		seed[i] = byte(i)
+	}
+
+	PolyDeriveUniformLeGamma1X4([4]*common.Poly{&ps[0], &ps[1], &ps[2], &ps[3]},
+		&seed, nonces)
+	for i := 0; i < 4; i++ {
+		PolyDeriveUniformLeGamma1(&p, &seed, nonces[i])
+		if ps[i] != p {
+			t.Fatalf("%d\n%v\n%v", i, p, ps[i])
```

Comment:
You may want to use `circl/internal/test.ReportError`

Reply:
Yeah, I added a reminder on #114 to fix this, not just for this PR but for the existing code as well.